core/config/project_settings.cpp

```diff
@@ -321,6 +326,9 @@ bool ProjectSettings::_set(const StringName &p_name, const Variant &p_value) {
 				autoload.path = path;
 			}
 			add_autoload(autoload);
+		} else if (p_name.operator String().begins_with("global_group/")) {
+			String group_name = p_name.operator String().split("/")[1];
```

Comment:
```suggestion
			String group_name = p_name.operator String().get_slice("/", 1);
```
The autoload code does this wrong. `get_slice()` is better here.